lib/Doctrine/DBAL/Schema/AbstractSchemaManager.php

```diff
@@ -941,27 +828,27 @@ protected function _getPortableUsersList($users)
     }
 
     /**
-     * @param string[] $user
+     * @param array<string, mixed> $user
      *
-     * @return string[]
+     * @return array<string, mixed>
      */
-    protected function _getPortableUserDefinition($user)
+    protected function _getPortableUserDefinition(array $user) : array
     {
         return $user;
     }
 
     /**
-     * @param mixed[][] $views
+     * @param array<int, array<string, mixed>> $views
      *
-     * @return View[]
+     * @return array<string, View>
      */
-    protected function _getPortableViewsList($views)
+    protected function _getPortableViewsList(array $views) : array
     {
         $list = [];
         foreach ($views as $value) {
             $view = $this->_getPortableViewDefinition($value);
 
-            if (! $view) {
```

Comment:
This doesn't look like a valid case. All platforms will create a view from the definition if they support views.

Reply:
I will see if I can get rid of this and remove the nullable on the method too.